### DISCUSSION TITLE
Can we deprecate Discord?

### DISCUSSION THREAD

**Discussion by nemobis** [mastodon/mastodon#20947] [General]
Some perks for Mastodon gGmbh donors and some development discussions [unfortunately](https://mako.cc/copyrighteous/free-software-needs-free-tools) rely on proprietary privacy-unfriendly messaging service Discord.

This is a problem, see also CNIL's announcement: [DISCORD INC. fined 800 000 euros](https://www.cnil.fr/en/discord-inc-fined-800-000-euros). It's clear that in an ideal world the Mastodon devs would never have used Discord in the first place, but we don't live in an ideal world.

It's not clear what can be done without overloading the development team further. The Discourse forum has been removed in the past due to the management overload it created (?) and the Matrix rooms are not used much. What else can be done to gradually move away from Discord (and later GitHub discussions too)? Maybe we can identify some workflow which doesn't work well in Discord and is currently taking a lot of time from the core Mastodon team, and identify an alternative that would require less efforts? Then it will become easier to think about replacing whatever it is that Discord is doing well.

**Comment by markcellus** [comment]
> The Discourse forum has been removed in the past due to the management overload it created 

I never quite understood the decision to remove Discourse forum. Not only does it require anyone who wants to participate to have a GitHub acccount. It also hands data over to GitHub, which is surprising given how privacy-oriented Mastodon is by its very nature.

**Reply by roke-julian-lockhart** [reply]
@markcellus, I agree that providing an asynchronous, FOSS forum would be an improvement, and that Discourse is the best available. However, *I* certainly don't determine which to utilise based upon such a vague notion as privacy, and would hope that Mastodon's developers don't, either.

**Comment by roke-julian-lockhart** [comment]
Does a Discord server exist for Mastodon?

**Reply by jeremiahlee** [reply]
Yes, access to official Mastodon Discord server is part of the Patreon sponsorship benefits. https://www.patreon.com/mastodon

**Comment by colinstu12** [comment]
Where is the official Mastodon Matrix server? If it’s still around?

**Reply by progval** [reply]
There has never been an official Mastodon Matrix server
